_src/_posts/2017-01-31-learning-rtb.markdown

```diff
@@ -0,0 +1,205 @@
+---
+layout: post
+title:  Learning from the RTB market
+date:   2017-01-31
+tag: data-science
+categories:
+  - data-science
+keywords: "python, RTB, machine-learning"
+author: memeplex
+---
+
+As you might already know, [Jampp][jampp] is a demand-side platform ([DSP][dsp])
+that allows companies to promote their mobile applications by leveraging
+real-time bidding ([RTB][rtb]) technologies in order to buy digital advertising
+from multiple inventory sources and exchanges. During an RTB transaction, an
+auction is announced and any interested bidder has to answer with a bid price
+within a time constraint of about 100 milliseconds. The bidder that wins the
+auction pays the second-highest price and obtains the right to print a creative
+(which is just jargon for displaying an ad) on a publisher site. If the banner
+is clicked by the user further goal events might be tracked, v.g. the
+application may be installed or opened. Obtaining a goal event is called a
+conversion.
+
+One main component of our platform is the machine learning subsystem that feeds
+the bidding subsystem with predictive models of conversion rates for different
+events of interest (clicks, installs, opens, etc.). A conversion rate for an
+event is the conditional probability of getting that event given the set of
+features that characterizes the current RTB transaction and given that we
+already printed a creative targeting that event. Why is it so important for our
```

Comment:
and given that we already printed -> and given that we have already printed
Anyway, the sentence is a bit confusing: is it a conditional probability after two events (each subclause introduced by "given") or are they two conditional probabilities (each "given" starts one). Perhaps the second "given" is not necessary, if the idea is to join the two events.